app/assets/javascripts/snowcrash/modules/issues/table.js.coffee

```diff
@@ -1,6 +1,6 @@
 class IssuesTable extends ItemsTable
   constructor: ->
-    super('issue')
+    super('#issues-table', 'issue')
```

Comment:
instead of hard-coding '#issues-table' this could be a parameter to the constructor.

Reply:
does this subclass need a special constructor or could we use some sort of "after_initialize" hook in the parent to add the tag/merge functionality?